/**:
  ros__parameters:
    object_separation_distance: 5.0  # [m]
